docs/guides/getting-started.md

```diff
@@ -73,9 +123,9 @@ Almost everything can be typed correctly without using `any`.
 If generic typing is not possible, you can use `unknown` or `object` for utility functions.
 In most other cases, the typings can be correctly applied.
 
-We encourage everybody to up their game about strict typing with TypeScript, but as always, there are ways around it.
-We supply a [`tsconfig.app-no-checks.json`](../../tsconfig.app-no-checks.json) in the source root, that can be used for Angular builders in the `angular.json` to disable all safety features.
-However, we don't support doing this for production builds.
+We encourage everybody to improve their skills in strict typing with TypeScript, but as always, there are ways around it.
```

Comment:
I would actually delete the second part of this sentence ", but as always, there are ways around it."

I also suggest deleting ll. 118 - 120 beginning with "Learning  the TypeScript ..." and ending with "... nobody will be the wiser." Doesn't match our tone of voice and is giving an opinion which is to be avoided. I also don't see any point in providing this kind of information. If I'm wrong, please rephrase this section.

Reply:
Changed as suggested.

Reply:
What about deleting the second part of this sentence ", but as always, there are ways around it."?